Include/String/String.h

```diff
@@ -11,7 +12,11 @@
 
 namespace PGE {
 
+#ifdef _WIN32
 typedef wchar_t wchar;
```

Comment:
Wouldn't we want to just use `char16_t` here?

Reply:
Yes, but all windows functions expect wchar_t so we'd need to do a ton of unnecessary casting. In the Windows compiler and OS wchar_t is fixed at 16 bits anyways, so we should be good.